corehq/pillows/mappings/xform_mapping.py

```diff
@@ -15,32 +15,32 @@
         "created": '2013-08-13',
     },
     "properties": {
-        'doc_type': {'type': 'string'},
+        'doc_type': {'type': 'text'},
```

Comment:
it would be great to convert all of these mappings to JSON files as we have for app and case mappings.

Reply:
I had this in my mind but I focused to fix errors in queries and mappings.